docs/rofl/app.mdx

```diff
@@ -241,6 +241,14 @@ services:
       - /run/rofl-appd.sock:/run/rofl-appd.sock
 ```
 
+:::warning Always specify FQDN image URL
+
+In contrast to Docker compose files, always use fully qualified domain name
```

Comment:
I would not say "In contrast to Docker compose files", but rather something like that this is to avoid security issues and ensure determinism, ROFL apps require fully qualified domain names. Because this is an explicit choice for security purposes, not related to Docker vs. Podman.